spec/features/merchants/merchant_items/show_spec.rb

```diff
@@ -144,4 +144,58 @@
       expect(page).to have_content("Current Price: $751.07")
     end
   end
+
+  it "There is a link to update the item that takes you to a form" do
+    visit merchant_item_path(@merchant_1, @item_1.id)
+
+    within("div#update-item") do
+      expect(page).to have_link("Update Item")
+    end
+
+    click_link("Update Item")
+
+    expect(current_path).to eq(edit_merchant_item_path(@merchant_1, @item_1.id))
```

Comment:
If we're testing that it takes us to a form, I feel like we should also use the capybara matchers `have_field` to test the proper fields on the edit merchant item path.

https://www.rubydoc.info/gems/capybara/Capybara%2FRSpecMatchers:have_field

I might add
```
within(div#edit_item_form) do
  expect(page).to have_field("Name")
  expect(page).to have_field("Description")
  expect(page).to have_field("Unit Price")
end
```